apps/token-manager/app/src/components/VestingInfoBoxes.js

```diff
@@ -0,0 +1,160 @@
+        >
+          <LocalIdentityBadge
+            entity={selectedHolder.receiver}
+            connectedAccount={isCurrentUser}
+          />
+        </div>
+        <div
+          css={`
+            padding: ${3 * GU}px;
+          `}
+        >
+          <InfoBoxHeader
+            icon={<TokenIcon />}
+            label={'Total Tokens'}
+            content={formatTokenAmount(
+              selectedHolder.holderBalance.balance,
+              18,
+              {
+                symbol: tokenSymbol,
+              }
+            )}
+          />
+        </div>
+      </Box>
+      <Box>
+        <InfoBoxHeader
+          icon={<VestingIcon />}
+          label={'Vested Tokens'}
+          content={formatTokenAmount(totalInfo.totalAmount, 18, {
+            symbol: tokenSymbol,
+          })}
+        />
+        <ul
+          css={`
+            margin-top: ${4 * GU}px;
+          `}
+        >
+          {[
+            [
+              'Unlocked',
+              <strong>
+                {formatTokenAmount(totalInfo.totalUnlocked, 18, {
+                  symbol: tokenSymbol,
+                })}
+              </strong>,
+            ],
+            [
+              'Locked',
+              <strong>
+                {formatTokenAmount(totalInfo.totalLocked, 18, {
+                  symbol: tokenSymbol,
+                })}
+              </strong>,
+            ],
+          ].map(([label, content], index) => (
+            <li
+              key={index}
+              css={`
+                display: flex;
+                justify-content: space-between;
+                list-style: none;
+                color: ${theme.surfaceContent};
+
+                > span:nth-child(1) {
+                  color: ${theme.surfaceContentSecondary};
+                }
+                > span:nth-child(2) {
+                  // “:” is here for accessibility reasons, we can hide it
+                  opacity: 0;
+                  width: 10px;
+                }
+                > span:nth-child(3) {
+                  flex-shrink: 1;
+                }
+                > strong {
+                  text-transform: uppercase;
+                }
+              `}
+            >
+              <span>{label}</span>
+              <span>:</span>
+              {content}
+            </li>
+          ))}
+        </ul>
+      </Box>
+      {isCurrentUser && (
+        <Box>
+          <InfoBoxHeader
+            icon={<TransferIcon />}
+            label={'Available to transfer'}
+            content={formatTokenAmount(totalInfo.totalUnlocked, 18, {
+              symbol: tokenSymbol,
+            })}
+          />
+        </Box>
+      )}
+    </React.Fragment>
+  )
+}
+function InfoBoxHeader({ icon, label, content }) {
+  const theme = useTheme()
+  return (
+    <div
+      css={`
+        display: flex;
+        justify-content: flex-start;
+      `}
+    >
+      {icon}
+      <div
+        css={`
+          display: inline-grid;
```

Comment:
Would `inline-flex` work here? It seems a bit weird specifying `grid` but then not setting any other grid values.